.github/workflows/inspektor-gadget.yml

```diff
@@ -1410,6 +1461,7 @@ jobs:
       - test-unit
       - build-clients
       - build-ig
+      - build-ig-cwe
       - build-gadget-container-images
       - build-helper-images
       - build-and-push-gadgets
```

Comment:
I'll propose to add `build-ig-cwe` only as a requirement for release. Otherwise it'll be very annoying to fix a CWE issue to later on find that other things as broken as well.